_clients/k8s-operator.md

```diff
@@ -0,0 +1,146 @@
+Then install the OpenSearch Kubernetes Operator using the following steps:
+
+1. In your preferred directory, clone the [OpenSearch Kubernetes Operator repo](https://github.com/Opster/opensearch-k8s-operator). Navigate into repo's directory using `cd`.
+2. Go to the `opensearch-operator` folder.
+3. Enter `make build manifests`.
+4. Start a Kubernetes cluster. When using minikube, open a new terminal window and enter `minikube start`. Kubernetes will now use a containerized minikube cluster with a namespace called `default`. Make sure that `~/.kube/config` points to the cluster.
+
+```yml
+apiVersion: v1
+clusters:
+- cluster:
+    certificate-authority: /Users/naarcha/.minikube/ca.crt
+    extensions:
+    - extension:
+        last-update: Mon, 29 Aug 2022 10:11:47 CDT
+        provider: minikube.sigs.k8s.io
+        version: v1.26.1
+      name: cluster_info
+    server: https://127.0.0.1:61661
+  name: minikube
+contexts:
+- context:
+    cluster: minikube
+    extensions:
+    - extension:
+        last-update: Mon, 29 Aug 2022 10:11:47 CDT
+        provider: minikube.sigs.k8s.io
+        version: v1.26.1
+      name: context_info
+    namespace: default
+    user: minikube
+  name: minikube
+current-context: minikube
+kind: Config
+preferences: {}
+users:
+- name: minikube
+  user:
+    client-certificate: /Users/naarcha/.minikube/profiles/minikube/client.crt
+    client-key: /Users/naarcha/.minikube/profiles/minikube/client.key
+```    
+   
+5. Enter `make install` to create the CustomResourceDefinition that runs in your Kubernetes cluster. 
+6. Start the OpenSearch Kubernetes Operator. Enter `make run`. 
+
+## Verify Kubernetes deployment
+
+To ensure that Kubernetes recognizes the OpenSearch Kubernetes Operator as a namespace, enter `k get ns | grep opensearch`. Both `opensearch` and `opensearch-operator-system` should appear as `Active`.
+
+With the operator active, use `k get pod -n opensearch-operator-system` to make sure that the operator's pods are running. 
+
+```
+NAME                                              READY   STATUS   RESTARTS   AGE
+opensearch-operator-controller-manager-<pod-id>   2/2     Running  0          25m
+```
+
+With the Kubernetes cluster running, you can now run OpenSearch inside the cluster.
+
+## Deploy a new OpenSearch cluster
+
+From your cloned OpenSearch Kubernetes Operator repo, navigate to the `opensearch-operator/examples` directory. There you'll find the `opensearch-cluster.yaml` file, which can be customized to the needs of your cluster, including the `clusterName` that acts as the namespace in which your new OpenSearch cluster will reside.
+
+With your cluster configured, run the `kubectl apply` command.
+
+```
+kubectl apply -f opensearch-cluster.yaml
+```
+
+The operator creates several pods, including a bootstrap pod, three OpenSearch cluster pods, and one Dashboards pod. To connect to your cluster, use the `port-forward` command.
+
+```
+kubectl port-forward svc/my-cluster-dashboards 5601
```

Comment:
while i hope that people using k8s are aware of this, it might be worth noting that this is only done for demo purposes; in a real setup an `Ingress` resource (or `Route` on OpenShift) must be set up to have a proper route available and not rely on port-forwarding